packages/amplication-client/src/Entity/EntityList.tsx

```diff
@@ -71,6 +71,7 @@ const EntityList: React.FC<Props> = ({ match, innerRoutes }) => {
   const { currentWorkspace, currentProject, currentResource } =
     useContext(AppContext);
 
+  const isResourceUnderLimitation = currentResource?.isUnderLimitation;
```

Comment:
Should we default this value to `false`? Otherwise we might have `disabled=undefined`